apps/heft/src/schemas/typescript.schema.json

```diff
@@ -58,6 +52,16 @@
       "type": "string"
     },
 
+    "buildProjectReferences": {
+      "description": "If true, enable behavior analogous to the \"tsc --build\" command. Will build projects referenced by the main project. Note that this will effectively enable \"noEmitOnError\".",
+      "type": "boolean"
+    },
+
+    "project": {
```

Comment:
I think `"tsconfigPath"` might make more sense.

Reply:
TypeScript, TSLint, and ESLint all refer to this as "project" (`--project` or `-p` in the CLI for the former two, "project" in .eslintrc for the latter). I'd prefer to be consistent.